app/src/main/java/com/stevesoltys/seedvault/restore/install/ApkRestore.kt

```diff
@@ -34,18 +38,29 @@ internal class ApkRestore(
     private val crypto: Crypto,
     private val splitCompatChecker: ApkSplitCompatibilityChecker,
     private val apkInstaller: ApkInstaller,
+    private val userManager: UserManager,
 ) {
 
     private val pm = context.packageManager
 
+    fun UserManager.checkRestriction(restriction: String): Boolean {
+        return this.userRestrictions.getBoolean(restriction, false)
+    }
+
+    private fun cantInstallApks(): Boolean {
+        return userManager.checkRestriction(DISALLOW_INSTALL_APPS) ||
+            userManager.checkRestriction(DISALLOW_INSTALL_UNKNOWN_SOURCES) ||
+            userManager.checkRestriction(DISALLOW_INSTALL_UNKNOWN_SOURCES_GLOBALLY)
```

Comment:
Would Seedvault be considered an unknown source? Why?

Reply:
Yes. Because the backup can come from anywhere.

This was discussed in the issue further.

One could craft a backup with just one app, get SeedVault to restore it, and thus bypass this restriction entirely.

We don't have a way of telling legit backups apart, so we leave it upto the system by letting it decide which app it wants to allow to be installed (depending upon the store configured), and which app it doesn't.